legal-api/src/legal_api/models/continuation_in.py

```diff
@@ -0,0 +1,59 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module holds all of the additional data about a continuation in."""
+from __future__ import annotations
+
+from .db import db
+
+
+class ContinuationIn(db.Model):  # pylint: disable=too-many-instance-attributes
+    """This class manages the continuation in."""
+
+    __versioned__ = {}
+    __tablename__ = 'continuation_ins'
+
+    id = db.Column(db.Integer, primary_key=True)
+    jurisdiction = db.Column('jurisdiction', db.String(10))
+    jurisdiction_region = db.Column('jurisdiction_region', db.String(10))
```

Comment:
Yes, 10 is plenty for country code (line 27).

Also, 10 is plenty for Canadian province or USA state codes.

However, there is some weird data in CPROD that we should either handle or clean up. See David Roberts' email I just forwarded you.